salt/beacons/inotify.py

```diff
@@ -45,6 +60,43 @@ def _get_notifier():
 def beacon(config):
     '''
     Watch the configured files
+
+    Example Config
+
+    .. code-block:: yaml
+
+        beacons:
+          inotify:
+            /path/to/file/or/dir:
+              mask:
+                - open
+                - create
+                - close_write
+              rec: True
```

Comment:
Would it make sense to make this:
```
recurse: True
```
That seems to match what we do elsewhere in salt when we want the function to recurse a directory structure.